### DISCUSSION TITLE
Pasting gets taken over in undesired ways

### DISCUSSION THREAD

**Discussion by akujbida** [keepassxreboot/keepassxc#9774] [Q&A] [answered]
When KeepassXC is open, I can't paste anything anywhere without having the paste interrupted and being sent back to KeepassXC. Really bizarre to me. Have I set something up incorrectly?

For example, after opening KeepassXC (not even necessary to open a db), every app where I copy then paste (EVERY app: geany, terminal, Excel, this window for typing this question, etc) the paste is blocked and screen goes back to KeepassXC. Highly undesirable.

Disabling browser integration yielded no change.

M1 Mac 13.5. Coming from Ubuntu keepassx (which I like very much, really enjoyed the copy paste method).
KeepassXC 2.7.6.
I have the Chrome extension but after uninstalling it, no change.

**Comment by droidmonkey** [comment]
Are you referring to actually copy/paste using the clipboard or Auto-Type? Not sure what browser integration has to do with this either. This would be a unique problem with your environment as that doesn't happen to anyone else as of now. Do you have some sort of custom clipboard manager installed?

**Reply by akujbida** [reply]
I'm a keyboard person, so for me copy paste means cmd-C then cmd-V.

I can't see how I'm unique. It's a new macbook Air M1, very few systems settings changed, no clipboard tools added. Pretty standard situation, seems to me.

I'm guessing there's a setting (or several) that by default cause this. I say default because I don't think I changed anything except as part of trying to fix this.

Wonderful to have a Maintainer in the conversation, thanks so much.

**Comment by michaelk83** [comment]
Sounds like maybe a keyboard shortcut conflict. Can you paste successfully using the target app's toolbar or menu? How exactly are you pasting when the problem occurs?

**Reply by akujbida** [reply]
Aha! Using the target app's Edit>Paste menu works for Chrome, Geany, Excel.

Does KeepassXC take over any system-wide keyboard shortcuts?

**Reply by droidmonkey** [reply, marked as answer]
Did you set your global Auto-Type shortcut to CMD +  V?

**Reply by akujbida** [reply]
Thanks very kindly for your attention, folks. I really liked the old one and am very keen to remain in the same system, so now I need just to re-read that user guide to make sure no more silly mistakes :)

To be specific about what I did: per droidmonkey, I changed the auto-type shortcut from cmd-V to some odd combination in this case cmd-shift-U.

I recall now having set it to cmd-V because I dislike having different shortcuts for each app but in this case it is required.
